e2e/playwright.config.ts

```diff
@@ -0,0 +1,29 @@
+import * as path from 'path'
+
+import { fileURLToPath } from 'url'
+
+import { defineConfig } from '@playwright/test'
+
+const __filename = fileURLToPath(import.meta.url)
+const __dirname = path.dirname(__filename)
```

Comment:
Also here we could use `import.meta` properties

Reply:
Fixed, thanks 👌